react/features/base/lib-jitsi-meet/functions.js

```diff
@@ -60,3 +60,45 @@ export function loadConfig(host: string, path: string = '/config.js') {
             throw err;
         });
 }
+
+/**
+ * Creates a JitsiLocalTrack for video from the given device id.
+ *
+ * @param {string} deviceId - The id of the target video source.
+ * @returns {Promise<JitsiLocalTrack>}
+ */
+export function createLocalVideoTrackPreview(deviceId) {
```

Comment:
this function is onlt used in one place, do we really need to export 2 of them? can't we export some `createLocalPreviewTrack` instead? Which then brings the question: there is nothing related to a preview here. It's just a local track. I think this could be simplidied.